### PR TITLE
Creating macros for validator syntax extenstion

### PR DESCRIPTION
**Pull request by PumpkinSeed**

### CODE REVIEW

**Comment by Cashmaney** [comment]
Very cool! @reuvenpo any comments?

File: cosmwasm/packages/wasmi-runtime/src/macros.rs [hunk 1]
```diff
@@ -51,3 +51,27 @@ macro_rules! coalesce {
         wrapper()
     }};
 }
+
+#[macro_export]
+macro_rules! validate_const_ptr {
+    ($ptr:expr, $ptr_len:expr) => {{
+        if let Err(_e) = validate_const_ptr($ptr, $ptr_len) {
```
Comment (reuvenpo) [inline cosmwasm/packages/wasmi-runtime/src/macros.rs:58]: `validate_const_ptr` should be fully qualified as `$crate::utils::validate_const_ptr`

File: cosmwasm/packages/wasmi-runtime/src/macros.rs [hunk 2]
```diff
@@ -51,3 +51,27 @@ macro_rules! coalesce {
...
+                ::enclave_ffi_types::EnclaveError::FailedFunctionCall,
+            ));
+        }
+    }};
+}
+
+#[macro_export]
+macro_rules! validate_mut_ptr {
+    ($ptr:expr, $ptr_len:expr) => {{
+        if let Err(_e) = validate_mut_ptr($ptr, $ptr_len) {
```
Comment (reuvenpo) [inline cosmwasm/packages/wasmi-runtime/src/macros.rs:70]: `validate_mut_ptr` should be fully qualified as `$crate::utils::validate_mut_ptr`

**Review by reuvenpo — COMMENTED** [review]
Hey, sorry, i forgot to reply to this after seeing a notification about your PR this morning. I only have one comment left :)

**Comment by PumpkinSeed** [comment]
I added the changes. :)

**Comment by reuvenpo** [comment]
I looked at the repo at your last commit, andwent ahead and applied some more changes.
I figured it would take less time than explaining what i wanted you to do :sweat_smile: 
Please review my last two commits, and let me know if you have any questions about what i did.
after you've reviewed them, we'll merge them to our repository :smile:

**Review by reuvenpo — APPROVED** [review]
Thanks for the PR @PumpkinSeed ! :smile:

**Pull request merged**